### ISSUE TITLE
winIcon doesn't replace the icon, merely adds a second one

### ISSUE THREAD

**Issue by ColonelBundy** [nwutils/nw-builder#274]
Resourehacker img: ![Resourehacker img](http://i.imgur.com/ZMzSxPB.png)
##### Issue:

Doesn't replace the nw.js icon. It simply adds another one after position 1 which means the nw.js icon will be present when creating shortcuts as first choice. This is an issue when using an installer etc. So in short terms the "winIcon" option is pretty useless unless fixed to actually force the icon to position 1.

**Comment by Forecaster** [comment]
I'm having this issue as well, sort of.
This happens to the win32 version, where it adds the icon, but the nw.js icon remains the primary.
But for win64 the icon is also added, but the custom icon appears instead of the nw.js icon for some reason. No idea what the difference is...

**Comment by BoatsAreRockable** [comment]
I am also experiencing this issue.

**Comment by MTRNord** [comment]
For me it shows in the Explorer the right one. On the Details the Default, on the Taskbar another and in The Topbar of The Exe when running again the default one.

**Comment by gitawego** [comment]
I've successfully made it work with resource hacker in command line mode
```shell
ICON_PATH=app\assets\favicon.ico
wine resource_hacker/ResourceHacker.exe --addoverwrite audit.exe, audit.exe, ${ICON_PATH},ICONGROUP,IDR_MAINFRAME,0,
``` 

I think we can replace module winresourcer by [node-resourcehacker](https://github.com/felicienfrancois/node-resourcehacker) to be able to benefit the resource hacker which  is better maintained.

**Comment by DblK** [comment]
I will commit a change soon for the windows Icon using rcedit instead of winresourcer.

**Comment by fbarrailla** [comment]
@DblK Hello, any update on this ?

**Comment by vankasteelj** [comment]
This works well for me, using a custom icon. I previously used resourcehacker (spawn in nodejs) but as of 3.5.6, it's working out of the box with nw-builder

**Comment by ayushmxn** [comment]
Fixed by #566
